src/utils/identity/ledger/simulated.ts

```diff
@@ -0,0 +1,52 @@
+import { PublicKey } from '@dfinity/agent';
+import { blobFromBuffer } from '@dfinity/candid';
+import { Principal } from '@dfinity/principal';
+import DfinityApp from '@zondax/ledger-icp';
+
+import Secp256k1PublicKey from '../secpk256k1/publicKey';
+import { Buffer as BlobBuffer } from '../../../../node_modules/buffer';
+import LedgerIdentity from '.';
+
+class SimulatedLedgerIdentity extends LedgerIdentity {
+
+  protected derivePath: string;
+  protected publicKey: PublicKey;
+  protected transport: any;
+  protected currentTx: { [key: string]: {signatureRs, } } = {};
+
+  public static async create(transport: any, derivePath = `m/44'/223'/0'/0/0`): Promise<SimulatedLedgerIdentity> {
```

Comment:
isnt this exactly the same as the default method from `LedgerIdentity`?

Reply:
Nop, here i am received transport, not TransportClass